docs/guides/StandarizedUnitsNaming.md

```diff
@@ -0,0 +1,115 @@
+
+- #### **Zulu** (głównodowodzący)
+
+- #### **Xray** (JFO - wsparcie Zulu <span style="color:red">???</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+
+- #### **Yankee** (NCO - wsparcie Zulu <span style="color:red">???</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+
+
+### <u>Siły lądowe (podstawa)</u>
+
+- #### **Alpha** (BPP)
+
+- #### **Bravo** (BPP)
+
+- #### **Charlie** (BPP)
+
+- #### **Delta** (BPP)
+
+
+### <u>Siły lądowe (wsparcie)</u>
+
+- #### **Golf** (wsparcie logistyczne)
+
+- #### **Juliett** (JTAC/FAC/ATC <span style="color:red">???</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+
+- #### **Kilo** (wsparcie pancerne)
+
+- #### **Papa** (wsparcie artyleryjskie) 
+
+
+### <u>Wojska specjalne</u>
+
+- #### **India** (komandosi)
+
+- #### **Lima** (snajperzy/rozpoznanie)
+
+
+### <u>Siły powietrzne</u>
+
+- #### **Mike** (transport)
+
+- #### **Oscar** (MEDEVAC)
+
+- #### **November** (CAS)
+
+
+
+## Uporządkowane wg alfabetu fonetyczego ICAO:
+
+- #### **Alpha** (BPP)
+- #### **Bravo** (BPP)
+- #### **Charlie** (BPP)
+- #### **Delta** (BPP)
+- #### **Echo** (<span style="color:red">???</span>) <!-- jakiś rodzaj specjalsów? prośba o wsparcie -->
+- #### **Foxtrot** (<span style="color:red">???</span>) <!-- jakiś rodzaj specjalsów? prośba o wsparcie --> 
+- #### **Golf** (wsparcie logistyczne)
+- #### **Hotel** (<span style="color:red">???</span>) <!-- nie pamiętam, prośba o wsparcie --> 
+- #### **India** (komandosi)
+- #### **Juliett** (JTAC/FAC/ATC <span style="color:red">???</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+- #### **Kilo** (wsparcie pancerne)
+- #### **Lima** (snajperzy/rozpoznanie)
+- #### **Mike** (transport)
+- #### **November** (CAS)
+- #### **Oscar** (MEDEVAC)
+- #### **Papa** (wsparcie artyleryjskie) 
+- #### **Quebec**(<span style="color:red">???</span>) <!-- nie pamiętam, prośba o wsparcie --> 
+- #### **Romeo** (<span style="color:red">???</span>) <!--  jakiś rodzaj specjalsów? nie pamiętam, prośba o wsparcie --> 
+- #### **Sierra** (<span style="color:red">???</span>) <!--  jakiś rodzaj specjalsów? nie pamiętam, prośba o wsparcie --> 
+- #### **Tango** (<span style="color:green">wolny</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+- #### **Uniform** (<span style="color:green">wolny</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+- #### **Victor** (<span style="color:green">wolny</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+- #### **Whiskey** (<span style="color:red">???</span>) <!-- nie pamiętam, prośba o wsparcie --> 
+- #### **Xray** (JFO - wsparcie Zulu <span style="color:red">???</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+- #### **Yankee** (NCO - wsparcie Zulu <span style="color:red">???</span>) <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+- #### **Zulu** (głównodowodzący)
```

Comment:
Please redo this section as a whole. After that we can discuss specific callsigns

1) Remove emphasis from headers
2) Remove all HTML. Being able to use it here is not a good reason for doing so
3) Don't use headings in Markdown lists, It's just pointless, only list will be effective anyway.
4) You're effectively repeating every callsign twice, consider if this could be done better.
5) Consider making this thing a table of `Znak wywoławczy` and `Opis`. If you wish, you could add a column for type of unit and keep it concise and self-explanatory.
6) You may leave unused callsigns without any description.

Reply:
Section re-done. Remarks 1-6 included. Good points over there, thanks!